Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -45,32 +56,50 @@ protected function configure()
             ->setDescription('Load data fixtures to your database.')
             ->addOption('fixtures', null, InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'The directory or file to load data fixtures from.')
             ->addOption('bundles', 'b', InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY, 'The bundles to load data fixtures from.')
+            ->addOption('services', null, InputOption::VALUE_NONE, 'Use services as fixtures')
+            ->addOption('group', null, InputOption::VALUE_IS_ARRAY|InputOption::VALUE_REQUIRED, 'Only load fixtures that belong to this group (use with --services)')
             ->addOption('append', null, InputOption::VALUE_NONE, 'Append the data fixtures instead of flushing the database first.')
             ->addOption('dm', null, InputOption::VALUE_REQUIRED, 'The document manager to use for this command.')
             ->setHelp(<<<EOT
-The <info>doctrine:mongodb:fixtures:load</info> command loads data fixtures from your bundles:
+The <info>doctrine:mongodb:fixtures:load</info> command loads data fixtures from your application:
 
-  <info>./app/console doctrine:mongodb:fixtures:load</info>
+  <info>php %command.full_name%</info>
 
 You can also optionally specify the path to fixtures with the <info>--fixtures</info> option:
+  <info>php %command.full_name%</info> --fixtures=/path/to/fixtures1 --fixtures=/path/to/fixtures2</info>
+
+If you want to use fixtures as services with autowiring, use <comment>--services</comment>
+Fixtures are services that are tagged with <comment>doctrine.fixture.odm</comment>.
+When using <comment>--services</comment>, both <comment>--bundles</comment> and <comment>--fixtures</comment> have no effect, 
+instead you can use the <comment>--group</comment> feature
 
-  <info>./app/console doctrine:mongodb:fixtures:load --fixtures=/path/to/fixtures1 --fixtures=/path/to/fixtures2</info>
+To execute only fixtures that live in a certain group (only available when using --services), use:
+
+  <info>php %command.full_name%</info> <comment>--group=group1</comment> <comment>--services</comment>
 
 If you want to append the fixtures instead of flushing the database first you can use the <info>--append</info> option:
 
-  <info>./app/console doctrine:mongodb:fixtures:load --append</info>
+  <info>php %command.full_name%</info> --append</info>
+
+
 EOT
         );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $dm = $this->getManagerRegistry()->getManager($input->getOption('dm'));
+        $ui = new SymfonyStyle($input, $output);
 
         $dirOrFile = $input->getOption('fixtures');
         $bundles = $input->getOption('bundles');
-        if ($bundles && $dirOrFile) {
-            throw new \InvalidArgumentException('Use only one option: --bundles or --fixtures.');
+        $services = $input->getOption('services');
+        if ($services === false) {
```

Comment:
I'd simplify the entire block:
```
if (($services && $bundles) || ($services && $dirOrFile) || ($bundles && $dirOrFile)) {
    throw new \InvalidArgumentException('Can only use one of "--bundles", "--fixtures", "--services".');
}
```

I don't think treating `--services` in combination with `--bundles` or `--fixtures` warrants a warning, I'd just throw a straight up error messages: chances are that people who decide to use `--services` already know what it does.